### PR TITLE
Add AudioTube

### PR DESCRIPTION
**Pull request by JBBgameich**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID. (Not directly me, but KDE does)
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by JBBgameich** [comment]
bot, build org.kde.audiotube

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: org.kde.audiotube.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    "modules": [
+        "generated/python3-yt-dlp.json",
+        "generated/python3-ytmusicapi.json",
+        {
+            "name": "pybind11",
+            "config-opts": ["-DPYBIND11_FINDPYTHON=ON", "-DPYBIND11_TEST=OFF"],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [ { "type": "git", "url": "https://github.com/pybind/pybind11", "tag": "v2.9.1" } ]
```
Comment (hfiguiere) [inline org.kde.audiotube.json:29]:
```suggestion
            "sources": [
                {
                    "type": "git", 
                    "url": "https://github.com/pybind/pybind11", 
                    "tag": "v2.9.1" 
                }
            ]
```

File: org.kde.audiotube.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        {
+            "name": "org.kde.audiotube",
+            "config-opts": ["-DPYBIND11_FINDPYTHON=ON"],
```
Comment (hfiguiere) [inline org.kde.audiotube.json:33]:
```suggestion
            "config-opts": [
                "-DPYBIND11_FINDPYTHON=ON"
            ],
```

File: org.kde.audiotube.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "generated/python3-ytmusicapi.json",
+        {
+            "name": "pybind11",
+            "config-opts": ["-DPYBIND11_FINDPYTHON=ON", "-DPYBIND11_TEST=OFF"],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [ { "type": "git", "url": "https://github.com/pybind/pybind11", "tag": "v2.9.1" } ]
+        },
+        {
+            "name": "org.kde.audiotube",
```
Comment (hfiguiere) [inline org.kde.audiotube.json:32]:
```suggestion
            "name": "audiotube",
```

File: org.kde.audiotube.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "/include/",
+        "/share/cmake/"
+    ],
+
+    "modules": [
+        "generated/python3-yt-dlp.json",
+        "generated/python3-ytmusicapi.json",
+        {
+            "name": "pybind11",
+            "config-opts": ["-DPYBIND11_FINDPYTHON=ON", "-DPYBIND11_TEST=OFF"],
```
Comment (hfiguiere) [inline org.kde.audiotube.json:26]:
```suggestion
            "config-opts": [
                "-DPYBIND11_FINDPYTHON=ON",
                "-DPYBIND11_TEST=OFF"
            ],
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by CarlSchwan** [comment]
Looks good to me

**Comment by JBBgameich** [comment]
bot, build org.kde.audiotube

**Comment by JBBgameich** [comment]
bot, build org.kde.audiotube

**Comment by JBBgameich** [comment]
This should be ready now from my side as far as I can see.

**Pull request closed without merging**